ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -97,7 +99,7 @@ public NetworkingModule(ReactApplicationContext reactContext, OkHttpClient clien
 
   @Override
   public void initialize() {
-    mClient.setCookieHandler(mCookieHandler);
+    // mClient.setCookieHandler(mCookieHandler); // TODO
```

Comment:
setCookieHandler has been removed. the new api is called CookieJar which is added when the OkHttpClient is built